apps/gui/gui-1.py

```diff
@@ -98,6 +106,14 @@ def _run( self, args ) :
 			Gaffer.NodeAlgo.applyUserDefaults( scriptNode )
 			self.root()["scripts"].addChild( scriptNode )
 
+		if args["vtune"].value :
```

Comment:
Any objections to dropping this for the gui app? I'm planning at some point to build support for enabling/disabling/visualising monitors directly in the GUI, and I think that's highly likely to conflict with this command line parameter.